sklearn/ensemble/bagging.py

```diff
@@ -298,13 +328,16 @@ def fit(self, X, y, sample_weight=None):
             for i in range(n_jobs))
 
         # Reduce
-        self.estimators_ = list(itertools.chain.from_iterable(
+        self.estimators_ += list(itertools.chain.from_iterable(
             t[0] for t in all_results))
-        self.estimators_samples_ = list(itertools.chain.from_iterable(
+        self.estimators_samples_ += list(itertools.chain.from_iterable(
             t[1] for t in all_results))
-        self.estimators_features_ = list(itertools.chain.from_iterable(
+        self.estimators_features_ += list(itertools.chain.from_iterable(
             t[2] for t in all_results))
 
+        if hasattr(self, "oob_score_") and self.warm_start:
+            del self.oob_score_
```

Comment:
I would do that around line 290-294 to handle fitted attribute at the same place.